### PR TITLE
go mod update

### PR DESCRIPTION
**Pull request by tharun-d**
we are using this package in production which has security vulnerabilities

*Description of changes:*
updating go mod packages and also go version

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,37 +1,92 @@
 module github.com/awslabs/aws-lambda-go-api-proxy
 
-go 1.14
+go 1.18
```
[the remark this answers is not in the document]
Comment (tharun-d) [reply go.mod]: Hi @drakejin updated to 1.20

[the remark this answers is not in the document]
Comment (drakejin) [reply go.mod]: Good! It's better!

**Review by drakejin — APPROVED** [review]

**Comment by tharun-d** [comment]
Hi @drakejin  @sapessi  can we get this merged ASAP, we have serious security issues in GIN package which will get resolved by this merge

**Comment by drakejin** [comment]
I have no right to merge. cc @sapessi  ..

**Comment by tharun-d** [comment]
@sapessi  can we merge this

**Comment by nicholas-keers** [comment]
Hello, I would love if you could merge this please @sapessi, the dependencies are starting to get some issues and the library has been left stale for a quite a few months now

**Pull request merged**